job_templates/restart_services_-_katello_ansible_default.erb

```diff
@@ -0,0 +1,18 @@
+<%#
+name: Restart Services - Katello Ansible Default
+snippet: false
+model: JobTemplate
+job_category: Katello
+description_format: Restart Services
+template_inputs:
+- name: helper
+  required: true
+  input_type: user
+  description: A comma separated list of commands to run to restart services
+  advanced: false
+provider_type: Ansible
+kind: job_template
+%>
+
+<%= render_template('Run Command - Ansible Default', :command => input(:helper).split(",").map {|split| "#{split}" }.join("\n")) %>
```

Comment:
this seems to produce invalid yaml as the run command contains `---` as the first line, result is (note --- in the middle)
```
---
- hosts: all
  tasks:
    - command: goferd
      register: out
    - debug: var=out
---
- hosts: all
  tasks:
    - command: katello-tracer-upload
      register: out
    - debug: var=out
```

Reply:
either it should be combined otherwise or this should be removed https://github.com/theforeman/community-templates/blob/develop/job_templates/run_command_-_ansible_default.erb#L17